v1/examples/inputs/pipeline_input_with_validator.yaml

```diff
@@ -0,0 +1,54 @@
+# For the following inputs, we haven't provided a value so, it's value will be asked during the runtime.
+#  Default value for input-1 is abc and for input-2, it is 1, so, if user doesn't provide a value during runtime,
+#  input-1 will be set as abc and input-2 will be set as 1.
+# Also, there's a validation that value should of input-1 should be one of a, b, abc
+#  for input-2, it should be a single digit number from 1 to 9
+
+version: 1
+kind: pipeline
+spec:
+  stages:
+    - name: test
+      id: test
+      description: <+inputs.input-1>
```

Comment:
this is invalid it needs to be either `<+inputs.input-1>` or `inputs.get("input-1")`